src/components/blocks/Modal.tsx

```diff
@@ -0,0 +1,18 @@
+import {useEffect} from "react";
+import { createPortal } from 'react-dom';
+
+const Modal = (props: any) => {
+    const el = document.createElement('div');
+    const modal = document.getElementById('modal') as HTMLHtmlElement;
+
+    useEffect(():any => {
+        modal.appendChild(el);
+        return () => modal.removeChild(el);
+    });
```

Comment:
Did you miss empty array here as dependencies?

Reply:
fixed it